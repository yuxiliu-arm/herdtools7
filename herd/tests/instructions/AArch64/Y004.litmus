AArch64 Y004
(* MUL *)

{
    0: X1 = 65536;
    0: X2 = 65536;
}

P0 ;
MUL W3, W1, W2;

forall (
    0: X1 = 65536 /\
    0: X2 = 65536 /\
    0: X3 = 0
)
