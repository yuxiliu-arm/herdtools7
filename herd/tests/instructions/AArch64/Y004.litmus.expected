Test Y004 Required
States 1
0:X1=65536; 0:X2=65536; 0:X3=0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=65536 /\ 0:X2=65536 /\ 0:X3=0)
Observation Y004 Always 1 0
Hash=64af9f2820eb24f19b9bd2665872394e

